frontend/src/network/notes_api.ts

```diff
@@ -1,11 +1,18 @@
 import { Note } from "../models/note";
+import { DeletedNote } from "../models/deletedNote";
 import { fetchData } from "./fetch_data";
 
 export async function fetchNotes(): Promise<Note[]> {
```

Comment:
you can also be descriptive here, `fetchLiveNotes` or `fetchExistingNotes` so that it reflects what you're doing inside the function better